### ISSUE TITLE
Auto-label bot fail on GH action

### ISSUE THREAD

**Issue by Cheukting** [Renato66/auto-label#25]
Got the following error in my build:

```
(node:1) UnhandledPromiseRejectionWarning: HttpError: Not Found
    at response.text.then.message (/node_modules/@octokit/request/dist-node/index.js:66:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
```

**Comment by Renato66** [comment]
@Cheukting thank's for the replyi will check it now!

**Comment by Renato66** [comment]
@Cheukting fixed it at v2 or v2.1.15

**Comment by Cheukting** [comment]
Amazing! Thanks!
